packages/mdc-top-app-bar/README.md

```diff
@@ -1,5 +1,21 @@
+<!--docs:
+title: "Top App Bar"
+layout: detail
+section: components
+excerpt: "A container for items such as application title, navigation icon, and action items."
+iconId: toolbar
+path: /catalog/top-app-bar/
+-->
+
 # Top App Bar
 
+<!--<div class="article__asset">
+  <a class="article__asset-link"
+     href="https://material-components-web.appspot.com/top-app-bar/index.html">
```

Comment:
We don't have a `top-app-bar` folder yet. We probably will in the future, but for now it's just `/top-app-bar.html`

Reply:
Oops. :facepalm: